File: harsha/Report/server/src/main/java/com/example/onetoonemapping/models/Report.java
package com.example.onetoonemapping.models;

import java.util.Date;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import org.hibernate.annotations.CreationTimestamp;
import lombok.Builder;
import lombok.Data;

@Entity
@Table(name = "reports")
@Builder
@Data
public class Report {

	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private int id;

	private String name;

	@Column(name = "createdAt", nullable = false, updatable = false)
	@CreationTimestamp
	private Date createdAt;

	public Report() {
	}

	public Report(int id, String name, Date createdAt) {
		this.id = id;
		this.name = name;
		this.createdAt = createdAt;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public Date getCreatedAt() {
		return createdAt;
	}

	public void setCreatedAt(Date createdAt) {
		this.createdAt = createdAt;
	}
}

File: Shubhankar/Report/server/src/models/report.ts
import {
  Column,
  CreateDateColumn,
  OneToMany,
  PrimaryGeneratedColumn,
} from "typeorm";
import { Reportpaneltest } from "./reportpaneltest";

export class Report {
  @PrimaryGeneratedColumn()
  id!: number;

  @Column()
  name?: string;

  @CreateDateColumn()
  createdAt!: Date;

  @OneToMany(
    (_type) => Reportpaneltest,
    (reportpaneltest: Reportpaneltest) => reportpaneltest.report_data
  )
  reportpaneltest!: Array<Reportpaneltest>;
}
